sockeye/model.py

```diff
@@ -45,6 +47,8 @@ class ModelConfig(Config):
     :param config_loss: Loss configuration.
     :param lexical_bias: Use lexical biases.
     :param learn_lexical_bias: Learn lexical biases during training.
+    :param weight_tying: Enables weight tying if True.
```

Comment:
I am not too happy with again adding more parameters to the model config here but I guess there is no other way...